### ISSUE TITLE
score_report.py: error: the following arguments are required: file

### ISSUE THREAD

**Issue by Pattie1995** [linguishi/chinese_sentiment#3]
你好，我在运行CNN目录下的score_report.py的时候，出现了如下的错误：
usage: score_report.py [-h] file
score_report.py: error: the following arguments are required: file
这个该如何解决呢？谢谢。
================================================
还有就是在训练CNN目录下的main.py的时候，最终的准确率非常低。
Saving dict for global step 16074: acc = 0.610625, global_step = 16074, loss = 0.6398508, precision = 0.5851781, recall = 0.76
Saving 'checkpoint_path' summary for global step 16074: results/model\model.ckpt-16074
Loss for final step: 0.586049.
